Packages/com.unity.inputsystem/InputSystem/Editor/InputActionAsset/CopyPasteUtility.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
```

Comment:
Everything that is editor-specific needs to have `#if UNITY_EDITOR ... #endif` around it. There's no automatic exclusion of editor stuff based on folder names in packages.

Reply:
fixed